llbuild.xcodeproj/project.pbxproj

```diff
@@ -2533,6 +2657,20 @@
 			shellPath = /bin/sh;
 			shellScript = "\"${SRCROOT}/utils/Xcode/create-target-link.sh\"";
 		};
+		BC8DEF2E20300F2000E9EF0C /* ShellScript */ = {
```

Comment:
I think it would be nicer to use a copy files phase for this, if possible.

Reply:
Didn't see an option for install dir in copy file phase :(

Reply:
Actually, let me see if copy file phase expands macro in Absolute Path option.

Reply:
Nah, can't use copy phase for files (.swiftmodule) inside BUILD_PRODUCT_DIR.